style.css

```diff
@@ -0,0 +1,34 @@
+body{
+    background-image: url("assets/background.png");
+    background-size: 100% 100%;
```

Comment:
This looks as though it might not be valid CSS :P.

Maybe background-size: 100%;  instead?